vaadin-date-time-picker-flow/src/main/java/com/vaadin/flow/component/datetimepicker/DateTimePicker.java

```diff
@@ -508,7 +571,7 @@ public void setMax(LocalDateTime max) {
      */
     public LocalDateTime getMax() {
         return PARSER.apply(
-                getElement().getProperty("max")
+            getElement().getProperty("max")
         );
     }
```

Comment:
Those properties are not synchronized with `@Synchronize()` and that's the reason why those are returning property of the `Element`. Otherwise it will work only with server side field and don't know anything about the changes on the client side. 
I am not sure if that should be changed.

Reply:
Afaik `getProperty()` does not make it fetch the current value from client side (it's not async). It only returns what the server already knows (either by synchronization or if the same property was previously set via `setProperty()` on the server side so I see no real benefit in using `getProperty()` precisely because it's not synchronized.

https://vaadin.com/docs/v14/flow/element-api/tutorial-properties-attributes.html (see first note block)
https://vaadin.com/docs/v14/flow/element-api/tutorial-user-input.html

I think currently in our components we're mainly using `@Synchronize()` for properties that may change due to normal user interaction (like "opened" state of a dialog/overlay).

Reply:
We can discuss this with others in the morning.